src/main/java/com/kaleido/fetch/service/ActivityService.java

```diff
@@ -46,8 +50,8 @@
     private String cabinetURI;
 
     public ActivityService(KaptureClient<Experiment> experimentKaptureClient, CabinetClient<PlateMap> cabinetClient) {
-		this.cabinetClient = cabinetClient;
-		this.experimentKaptureClient = experimentKaptureClient;
+		    this.cabinetClient = cabinetClient;
```

Comment:
Can you do a find and replace of tab with space, it appears that there is a tab in the line before the spaces that you added.

Reply:
Replaced tab with spaces